timezone/src/include/timezone/dialogs.rb

```diff
@@ -573,21 +573,18 @@ def TimezoneDialog(args)
         # true by default (fate#303520)
         @ntp_used = true
         # configure NTP client
-        Builtins.srandom
-        @ntp_server = Builtins.sformat(
-          "%1.opensuse.pool.ntp.org",
-          Builtins.random(4)
-        )
+        # to prevent misusage of ntp.org we need to distinguish opensuse and SLE usage
+        base_products = Product.FindBaseProducts
```

Comment:
There is a missing `Yast.import "Product"` at the top.

Reply:
Good catch!